chemprop/models/mpn.py

```diff
@@ -195,6 +195,8 @@ def forward(self,
 
         :param batch: A list of list of SMILES, a list of list of RDKit molecules, or a
                       list of :class:`~chemprop.features.featurization.BatchMolGraph`.
+                      The outer list is of length :code:`number_of_molecules`, the inner list or
+                      BatchMolGraph is of length :code:`num_molecules` (number of datapoints in batch).
```

Comment:
Could use a parenthetical saying the outer list is the number of molecules per datapoint.

Reply:
Thank you for reviewing!

Regarding the lists: In the forward() function in chemprop/models/mpn.py a batch is generated by calling mol2graph on each item of the outer list [see here](https://github.com/chemprop/chemprop/blob/339b6e647f413c7bf656c1d76dfa4b88f88acf1b/chemprop/models/mpn.py#L227) I think this only makes sense if the outer list iterates over the multi-molecule input, and the inner list over the datapoints in the batch, since mol2graph generates a BatchMolGraph object, which featurizes a list of molecules and zero-pads them (which is necessary for the list of molecules in the batch (=datapoints) but not the list of molecules in the multi-molecule input. Also, later in the function we encode each element in the outer list with a separate encoder in self.encoder (of length number_of_molecules = multi-molecule number). I thus assumed the outer list corresponds to multi-molecule, and the inner list to the batch size.

 I furthermore looked at PR76 (when we merged in multi-molecule support) and one of the changes in chemprop/data/data.py was to make batch_graph() output a list of BatchMolGraph objects (one for each molecule in each smiles input line) - supporting my line of thought, [see here](https://github.com/chemprop/chemprop/blob/339b6e647f413c7bf656c1d76dfa4b88f88acf1b/chemprop/data/data.py#L253-L264). 

So, I'm say 99% sure that the outer list is the number of multi-molecules - but I wasn't involved in that code change at all, so I might be missing something (hopefully not).
 
 I will include your suggestion to the docstring!